databricks-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/databricks_run.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import logging
+
+from kedro.framework.project import configure_project
+from kedro.framework.session import KedroSession
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--env", dest="env", type=str)
+    parser.add_argument("--conf-source", dest="conf_source", type=str)
+    parser.add_argument("--package-name", dest="package_name", type=str)
+
+    args = parser.parse_args()
+    env = args.env
+    conf_source = args.conf_source
+    package_name = args.package_name
+
+    # https://kb.databricks.com/notebooks/cmd-c-on-object-id-p0.html
+    logging.getLogger("py4j.java_gateway").setLevel(logging.ERROR)
+    logging.getLogger("py4j.py4j.clientserver").setLevel(logging.ERROR)
+
+    configure_project(package_name)
+    with KedroSession.create(env=env, conf_source=conf_source) as session:
+        session.run()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
How does this CLI expected to be run? So `kedro run` won't be used on Databricks?

Reply:
`kedro run` can't be used on Databricks because of Click: https://github.com/kedro-org/kedro/issues/1807.

This script is a way around that for now.

Reply:
Wow, this was a good rabbit hole... Good that we have a workaround for now.